client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -135,34 +143,131 @@ class DailyBackupStatus extends Component {
 						{ translate( 'Contact support' ) }
 					</Button>
 				</div>
-			</Card>
+			</>
 		);
 	}
 
-	renderNoBackups() {
-		const { translate } = this.props;
+	renderNoBackup() {
+		const { translate, selectedDate, onDateChange } = this.props;
 
-		// todo: remove the mock dates by the real dates
-		const lastBackupAt = 'Yesterday 16:02';
-		const nextBackupAt = 'Today 16:02';
+		const displayDate = selectedDate.format( 'll' );
+		const nextDate = selectedDate.clone().add( 1, 'days' );
+		const displayNextDate = nextDate.format( 'll' );
 
 		return (
-			<Fragment>
-				<Gridicon icon="sync" />
+			<>
+				<Gridicon icon="cloud-upload" className="daily-backup-status__gridicon-no-backup" />
+				<div className="daily-backup-status__title">{ translate( 'No backup' ) }</div>
 
-				<div className="daily-backup-status__date">
-					{ translate( 'Backup Scheduled' ) }: { nextBackupAt }
+				<div className="daily-backup-status__label">
+					<p>
+						{ translate( 'The backup attempt for %(displayDate)s was delayed.', {
+							args: { displayDate },
+						} ) }
+					</p>
+					<p>
+						{ translate(
+							'But don’t worry, it was likely completed in the early hours the next morning. ' +
+								'Check the following day, {{link}}%(displayNextDate)s{{/link}} or contact support if you still need help.',
+							{
+								args: { displayNextDate },
+								components: {
+									//todo: href need implementation and add the correct query
+									link: (
+										<a
+											href="?date="
+											onClick={ event => {
+												event.preventDefault();
+												onDateChange( nextDate );
+											} }
+										/>
+									),
+								},
+							}
+						) }
+					</p>
 				</div>
-				<div>{ translate( 'Last daily backup' ) + ` ${ lastBackupAt }` }</div>
 
+				<Button
+					className="daily-backup-status__support-button"
+					href="https://jetpack.com/contact-support/"
+					target="_blank"
+					rel="noopener noreferrer"
+					isPrimary={ false }
+				>
+					{ translate( 'Contact support' ) }
+				</Button>
+			</>
+		);
+	}
+
+	renderNoBackupToday( lastBackupDate ) {
+		const { translate, timezone, gmtOffset, moment, onDateChange } = this.props;
+
+		const today = applySiteOffset( moment(), {
+			timezone: timezone,
+			gmtOffset: gmtOffset,
+		} );
+		const yesterday = today.subtract( 1, 'days' );
+
+		const lastBackupDay = lastBackupDate.isSame( yesterday, 'day' )
+			? translate( 'Yesterday ' )
+			: lastBackupDate.format( 'll' );
+
+		const lastBackupTime = lastBackupDate.format( 'LT' );
+
+		// Calculates the remaining hours for the next backup + 3 hours of safety margin
+		const hoursForNextBackup =
+			parseInt( lastBackupDate.format( 'H' ) ) - parseInt( today.format( 'H' ) ) + 3;
+
+		return (
+			<>
+				<Gridicon className="daily-backup-status__gridicon-backup-scheduled" icon="cloud-upload" />
+				<div className="daily-backup-status__static-title">
+					{ translate( 'Backup Scheduled:' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'Backup Scheduled' )` *ES Score: 11*